README.md

```diff
@@ -1,2 +1,17 @@
 # web3-react-native-dapp-viem
-A modern Web3 dApp with React Native, viem and WalletConnectModal
+
+This repository showcases how to build a modern web3 dApp with React Native, [viem](https://viem.sh) and [WalletConnectModal](https://docs.walletconnect.com/advanced/walletconnectmodal/about)
+
+## Requirements
+
+- [Expo environment setup](https://docs.expo.dev/get-started/installation/#requirements) (Node.js, Git, Watchman)
+- A [Wallet Connect Cloud](https://cloud.walletconnect.com/sign-in) project ID
+- Expo Go app installed in your smartphone
+- One or more web3 wallets installed in your smartphone (e.g. MetaMask, Rainbow Wallet, Trust Wallet, etc)
+
+## How to run
+
+- In `App.tsx` fill in `projectId="..."` with your Wallet Connect Cloud project ID
+- `npm start`
```

Comment:
[nit] Could we go with `yarn`?

Reply:
@mdjastrzebski I think I'd prefer staying with `npm` as it's the default. There are a lot of new web3 devs that are not fully familiarized with the JS ecosystem and might not know what is yarn. Also, Yarn PnP (v4) has breaking changes and needs some massaging to work with RN

Reply:
Personally I would go with yarn 1 (we use it in all our project where I work at CK), but this is minor and we can stay with NPM.